services/gateway-api/src/circuit_breaker/health_monitor.py

```
@@ -0,0 +1,450 @@
+    async def stop(self):
+        """Stop health monitoring"""
+        self._running = False
+        if self._monitoring_task:
+            self._monitoring_task.cancel()
+            try:
+                await self._monitoring_task
+            except asyncio.CancelledError:
+                pass
+        logger.info("Health monitoring stopped")
+
+    async def _monitoring_loop(self):
```

Ruff (0.14.8)

Missing return type annotation for private function `_monitoring_loop`

Add return type annotation: `None`

(ANN202)